src/layout/Dashboard.tsx

```diff
@@ -33,7 +29,6 @@ const Dashboard = (props: Props): ReactElement => {
       <Container className={classes.content}>
         <ErrorBoundary>
           <>
-            <AlertVersion />
```

Comment:
What should we do about AlertVersion?

Reply:
IMHO it should stay in general but we should make it less strict.